go.mod

```diff
@@ -25,6 +25,7 @@ require (
 )
 
 require (
+	github.com/Masterminds/semver/v3 v3.2.1 // indirect
```

Comment:
might want to consider this instead https://pkg.go.dev/golang.org/x/mod/semver

Reply:
They use "v0.0.0" instead of regular semantic versioning, not sure if we want to include that or not

Reply:
@mxyng beat me to it. Please use that semver package. It's what we're using in the registry and we should keep our semver comparison logic in sync.

Reply:
@joshyan1 I should add, Ollama can (and should IMO) add the `v` prefix and not require, or even allow, the v in the value of an `OLLAMA` pragma.